### DISCUSSION TITLE
is there a way to have one minidexed DX7 on a Raspberry PI Pico?

### DISCUSSION THREAD

**Discussion by DutchBigEd** [probonopd/MiniDexed#431] [Q&A] [answered]
I have this old midi keyboard laying around and would love to have **just one instance** of a minidexed DX7 built in it.
Is the Raspberryu PI Pico RP2040, dual-core 264kb cortex m0 processor capable to pull this off?
And how should I do this if this is possible? Since the Pico lacks a uSD card connector...
I do have an PCM5102 audio hat.

Thanks,
Ed

edit: text

**Comment by BobanSpasic** [comment, marked as answer]
Pico is not compatible with the rest of the Pi series. If you want a low-cost solution, you could go for Pi Zero.

Edit: - I forgot - you could also go for the Teensy board and MicroDexed: https://www.parasitstudio.de/
